packages/framework/assets/js/admin/components/sideMenu.js

```diff
@@ -4,12 +4,23 @@ import Register from '../../common/utils/register';
 export default class SideMenu {
 
     constructor ($sideMenu) {
+        this.$webPanel = $('.js-web-panel');
         this.$sideMenu = $sideMenu;
         this.$items = this.$sideMenu.filterAllNodes('.js-side-menu-item');
-
-        this.$items.click(function () {
-            $(this).filterAllNodes('.js-side-menu-submenu').show();
-            $(this).addClass('open');
+        const _this = this;
+
+        this.$items.click( event => {
+            if($(event.currentTarget).hasClass('open')){
```

Comment:
I would recommend create from `open`, `hidden`, `.js-side-menu-submenu` and `.js-side-menu-item` strings constants

Reply:
open and hidden are state class ... like .active we usually use them - and they are important for using css attributes.

Reply:
A understand, but my point was about it that you use `open` string by eight times, `hidden` and `.js-side-menu-submenu`  by four times and `.js-side-menu-item` by two times. And it is weird for me :smile: But when you vouch for it will never be changed ...